lib/Doctrine/ORM/Persisters/Entity/EntityPersister.php

```diff
@@ -114,12 +114,12 @@ public function addInsert($entity);
      *
      * If no inserts are queued, invoking this method is a NOOP.
      *
-     * @psalm-return list<array{
+     * @psalm-return void|list<array{
      *                   generatedId: int,
      *                   entity: object
-     *               }> An array of any generated post-insert IDs. This will be
+     *               }> Returning an array of any generated post-insert IDs. This will be
```

Comment:
Why did you feel the need to add "Returning an" here? It feels like you wanted to write "Returning an array of… is deprecated", but stopped halfway.

Reply:
fixed